site/source/community/user-groups.html.md

```diff
@@ -44,6 +44,7 @@ Mesos User Groups (MUGs) are responsible for organizing local meetups, hackathon
 * [New York, NY](http://www.meetup.com/Apache-Mesos-NYC-Meetup/)
 * [San Francisco, CA](http://www.meetup.com/Bay-Area-Mesos-User-Group/)
 * [Seattle, WA](http://www.meetup.com/Seattle-Mesos-Meetup/)
+* [Raleigh/Durham, NC] (http://www.meetup.com/Triangle-Area-Mesos-User-Group/)
```

Comment:
Looks good to me. Could you put it after `[New York, NY]`, because it is sorted by alphabetically. By the way, you need find a committer help you submit this.